forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/utils/ClaimsUtils.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2019 ForgeRock AS.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.forgerock.openbanking.common.utils;
+
+import com.forgerock.openbanking.constants.OIDCConstants;
+import com.forgerock.openbanking.model.claim.Claims;
+import com.nimbusds.jose.shaded.json.JSONObject;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
+
+import java.text.ParseException;
+import java.util.Map;
+
+/**
+ * Utility method(s) for manipulating a JWT's claims
+ */
+public class ClaimsUtils {
```

Comment:
Just `Claims` or should this be `JwtUtils`? :-)

Reply:
I don't know where this class will go. No problem to refactor it later if non-claims jwt goes in there.